google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfiguration.java

```diff
@@ -61,13 +41,14 @@ public String toString() {
   private String environment;
 
   private String dockerFilePath;
-  private String appYamlPath;
+  private String yamlPath;
```

Comment:
I have an issue with asking users to specify an app.yaml file when they can specify any yaml file that's not called app.yaml.

I can rename the variables and internal stuff to appYaml but keep the UI as YAML?